### DISCUSSION TITLE
Illustration resizing request

### DISCUSSION THREAD

**Discussion by kasey-seo** [twilio-labs/paste#2074] [Request: Illustration] [answered]
Hi team,

Can I please request resizing for the illustrations below? 
They will be added to Figma.
Thank you in advance!

- Enterprise https://library.twilio.com/share/0B06AF02-1A85-41D6-8C712765E741B791/?mediaId=179169BC-21FD-498D-B8CF39A3A72DD682
- Campaign https://library.twilio.com/share/0B06AF02-1A85-41D6-8C712765E741B791/?mediaId=D38DB400-765F-4D87-B4E6DC36264AFF68
- Awaiting data https://library.twilio.com/share/0B06AF02-1A85-41D6-8C712765E741B791/?mediaId=CCE495BC-0D72-4F3D-A7E4958ECA83A41B
- Startup https://library.twilio.com/share/0B06AF02-1A85-41D6-8C712765E741B791/?mediaId=1E391BB9-29BC-41CE-965C91916B031833

**Comment by kylegilbreath** [comment, marked as answer]
Hi Kasey! Thanks for the request. I've created a ticket on our backlog to get these illustrations added to the Figma library and to dedicated product UI Bynder folder. I'll follow up when we get this request completed.

**Reply by kylegilbreath** [reply]
Hi @kasey-seo! I just wanted to follow up and let you know that we've added the illustrations to our Figma library and to the Bynder folders. You can find the illustrations here:

Enterprise -> Brand Illustrations/Marketing
SendGrid Campaigns -> Brand Illustrations/Product branding
Awaiting data -> Brand Illustrations/UI
Startup -> Brand Illustrations/Marketing

Let us know if you have any issues accessing these!
